styles/Button.module.scss

```diff
@@ -0,0 +1,20 @@
+.socialBtn {
+  @apply flex gap-4;
```

Comment:
I don't think grid gap does anything here yet, but it might when we take into consideration the changes for icon content btns.